packages/cozy-client/src/CozyClient.js

```diff
@@ -1081,6 +1081,17 @@ client.query(Q('io.cozy.bills'))`)
     return this.hydrateDocument(obj)
   }
 
+  /**
+   * Creates an association that is linked to the store.
+   */
+  getAssociation(document, associationName) {
+    return createAssociation(
+      document,
+      this.schema.getAssociation(document._type, associationName),
+      this.getRelationshipStoreAccessors()
+    )
+  }
+
```

Comment:
Why this revert?

Reply:
Because the cozy bar apparently rely on getAssociation. Instead of removing it, I am fixing it in the next commit.